Gruntfile.js

```diff
@@ -39,144 +57,181 @@ module.exports = function( grunt ) {
           "unqualified-attributes": false,
           "vendor-prefix": false,
           "zero-units": false
-        },
-        src: [
-          "public/learning_projects/**/*.css"
-        ]
+        }
       }
     },
-    lesslint: {
-      src: ["public/stylesheets/userbar.less"],
+    jshint: {
+      server: {
         options: {
-          csslint: {
-            "adjoining-classes": false,
-            "box-model": false,
-            "box-sizing": false,
-            "bulletproof-font-face": false,
-            "compatible-vendor-prefixes": false,
-            "floats": false,
-            "font-sizes": false,
-            "ids": false,
-            "important": false,
-            "outline-none": false,
-            "overqualified-elements": false,
-            "qualified-headings": false,
-            "regex-selectors": false,
-            "star-property-hack": false,
-            "underscore-property-hack": false,
-            "universal-selector": false,
-            "unique-headings": false,
-            "unqualified-attributes": false,
-            "vendor-prefix": false,
-            "zero-units": false
-          }
+          "-W069": true, // ignore "['...'] is better written in dot notation." warnings
+          // Enforcing
+          bitwise       : false,     // true: Prohibit bitwise operators (&, |, ^, etc.)
```

Comment:
Settings are different between front and back end code. Would it be better to unify them?